src/assets/stylesheets/2d-hud.scss

```diff
@@ -6,6 +6,7 @@
   align-items: center;
   height: 80px;
   width: 100%;
+  user-select: none;
```

Comment:
i think i've concluded that there is an infinite number of CSS attributes I do not know about